src/modal/modal.ts

```diff
@@ -21,7 +21,7 @@ export interface NgbModalOptions {
   /**
    * Size of a new modal window.
    */
-  size?: 'sm' | 'lg';
+  size?: 'sm' | 'lg' | 'md';
```

Comment:
I also agree with **jnizet** that adding "md" is unnecessary. It's how everything works in general. Eg. all controls have a default size but you can add "lg" or "sm" classes. However, there is no "md" setting as logically that is the default.